ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -154,44 +165,72 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
     return;
   }
 
+  setTierRequirement(membershipRequirement, attribute) {
+    // TODO(RW-6933): Implement new institution admin UI with CT support.
+    this.setState(fp.set(['institution', attribute],
+      [{accessTierShortName: AccessTierShortNames.Registered,
```

Comment:
nit: this wrapping is a bit tough to read, I suggest putting each key on one line if you can't fit the whole object into a single line